apps/webbsite/src/styles/globals.css

```diff
@@ -57,67 +54,169 @@
   @apply !bg-mono-200;
 }
 
-/** Headings and subheadings */
-
+/* Headings, sub-headings, card texts */
 .heading-1 {
-  @apply text-[32px] leading-[46px] tracking-tight font-bold text-mono-200 font-satoshi
-    md:text-[84px] md:leading-[96px] md:tracking-normal;
+  @apply text-base
+  text-[34px] leading-[45.9px] tracking-[-0.011em] /** small screen */
+  md:text-[84px] md:leading-[113px] /** medium screen */
+  font-black font-satoshi;
 }
 
 .heading-2 {
-  @apply text-[30px] leading-[41px] text-mono-200 font-bold font-satoshi
-    md:text-[48px] md:leading-[72px];
+  @apply text-base
+  text-[30px] leading-[40.5px] /** small screen */
+  md:text-[48px] md:leading-[72px] /** medium screen */
+  font-black font-satoshi;
 }
 
 .heading-3 {
-  @apply text-[24px] leading-8 text-mono-200 font-black font-satoshi
-    md:text-[36px] md:leading-[54px];
+  @apply text-base
+  text-[24px] leading-[32.4px] /** small screen */
+  md:text-[36px] md:leading-[54px] /** medium screen */
+  font-black font-satoshi;
 }
 
 .heading-4 {
-  @apply text-[24px] leading-[36px] text-mono-200 font-black font-satoshi;
+  @apply text-base
+  text-[24px] leading-[36px] /** small screen */
+  font-bold font-satoshi;
 }
 
 .sub-heading-1 {
-  @apply text-base font-medium text-mono-180 font-satoshi md:text-2xl md:leading-9;
+  @apply text-base
+  text-[16px] leading-[21.6px] /** small screen */
+  md:text-[18px] md:leading-[27px] /** medium screen */
+  font-medium font-satoshi;
 }
 
 .sub-heading-2 {
-  @apply text-base leading-[22px] font-medium text-mono-180 font-satoshi
-    md:text-[18px] md:leading-[27px];
+  @apply text-base
+  text-[16px] leading-[21.6px] /** small screen */
+  md:leading-[24px] /** medium screen */
+  font-medium font-satoshi;
+}
+
+.button-primary {
```

Comment:
Can you talk more about why you want a custom class name for the primary button and secondary button here, why don't we use the `Button` component from `webb-ui-kit`, the below class is mostly the same with the `Button` component.

Reply:
@AtelyPham You're right we can just use `Button` component. I'm currently passing `button-primary` and `button-secondary` classname to the `Button` components. We need to override default `Button` styles to match with the design. For example text size, padding is different on different screen sizes in `webbsite` compared to `stats` and `bridge`. Instead of passing overriding classnames to match with the design to all the `Button`'s we're using in `webbsite`, I've created a single class.

Example:

`Button` without overriding class names.

<img width="204" alt="Screenshot 2023-03-02 at 7 23 36 AM" src="https://user-images.githubusercontent.com/53374218/222471674-08b64ed7-2d39-458a-9022-e7e50837832c.png">

<img width="259" alt="Screenshot 2023-03-02 at 7 15 59 AM" src="https://user-images.githubusercontent.com/53374218/222471797-fd18dcd5-cbef-4ab0-bd49-5c49f338282f.png">

`Button` with `primary-button` class name.

<img width="234" alt="Screenshot 2023-03-02 at 7 25 09 AM" src="https://user-images.githubusercontent.com/53374218/222472065-2d1dee7b-194b-4a42-bb0b-52f8f265730d.png">

<img width="251" alt="Screenshot 2023-03-02 at 7 16 11 AM" src="https://user-images.githubusercontent.com/53374218/222472110-ba69350b-b1d5-4b57-91bc-681c8cc2bb9a.png">

I'm not sure if you can see the subtle difference in the screenshot but there is.